### PR TITLE
Shadow Dimension visual pack

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR

shadow tree
chromite floor
shadow basalt
fix gold ore on new walls

## Why / Balance

for Emo 

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
no cl

### CODE REVIEW

File: Resources/Textures/Tiles/attributions.yml [hunk 1]
```diff
@@ -95,3 +95,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "taken at https://github.com/ParadiseSS13/Paradise/blob/8b7f4c8b69c74c6de5a755272eb8d3520f3d87c7/icons/turf/floors.dmi"
   source: "https://github.com/ParadiseSS13/Paradise"
+  
+- files: ["chromite.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "taken at commit 0587dd16e28108bdf0b0a28e2caae4319845e861, and recolored by TheShuEd"
+  source: "https://github.com/Mojave-Sun/mojave-sun-13"
```
Comment (metalgearsloth) [inline Resources/Textures/Tiles/attributions.yml:102]: Is this the right commit you used?

File: Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "taken from https://github.com/tgstation/tgstation/raw/master/icons/turf/floors.dmi and edited by TheShuEd",
+  "size": {
```
Comment (metalgearsloth) [inline Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json:5]: Needs the commit used.

File: Resources/Textures/Objects/Decoration/Flora/flora_shadow_trees.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/e00cae8d065f9cf520688cc0dd0e15ba5bef12a9 and recolor by TheShuEd",
+    "size": {
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Decoration/Flora/flora_shadow_trees.rsi/meta.json:5]: Needs an up to date commit that was used to get.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Textures/Tiles/attributions.yml (hunk 1 above)
Comment (TheShuEd) [reply Resources/Textures/Tiles/attributions.yml:102]: link copied from the original texture in ss14 
then recolored

File: Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json (hunk 2 above)
Comment (TheShuEd) [reply Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json:5]: link copied from the original texture in ss14

File: Resources/Textures/Objects/Decoration/Flora/flora_shadow_trees.rsi/meta.json (hunk 3 above)
Comment (TheShuEd) [reply Resources/Textures/Objects/Decoration/Flora/flora_shadow_trees.rsi/meta.json:5]: https://github.com/tgstation/tgstation/blob/e00cae8d065f9cf520688cc0dd0e15ba5bef12a9/icons/obj/flora/jungleflora.dmi
this link?

File: Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json (hunk 2 above)
Comment (metalgearsloth) [reply Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json:5]: It's linking to tg's master which isn't valid, it should reference the commit it's pulled from please.

File: Resources/Textures/Objects/Decoration/Flora/flora_shadow_trees.rsi/meta.json (hunk 3 above)
Comment (metalgearsloth) [reply Resources/Textures/Objects/Decoration/Flora/flora_shadow_trees.rsi/meta.json:5]: yeah that looks good.

File: Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json (hunk 2 above)
Comment (TheShuEd) [reply Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json:5]: problem: can't find this texture in tg build. It seems that they moved it somewhere very much, I've searched many files, but still can't find it.

Comment (metalgearsloth) [inline Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json:5]:
```suggestion
  "copyright": "taken from tgstation @ commit a0ca7b3f46132517f71f08bfda465667d133b5d7 and edited by TheShuEd",
  "size": {
```

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**